storybook/header-preset/header.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import { render } from "react-dom"
+import {
+  Link as NavLink,
+  NavigationBar,
+} from "../../packages/component-library/components/NavigationBar"
+
+import Footer from "../../site/src/components/Footer"
+
+const styles = require("./header.module.scss")
+
+export const SiteHeader = () => {
```

Comment:
I considered moving this component to be inside the `./site` directory. If we ever need to update the navigation content we'll need to remember to update it here too. I used the existing Kaizen component, so we don't need to worry about syncing visual changes.

Reply:
This changes pretty infrequently — maybe a big ole comment at the top of each file, warning of the duplication?

Reply:
Updated :D